tests/integration/targets/letsencrypt/runme.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
where is this used?

Reply:
if there is a file "runme.sh" in the integration-tests folder, `ansible-test` automatically runs it.